CHANGELOG.md

```diff
@@ -1,14 +1,38 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Add type safe methods for:
+    - `RealmCollection.min(ofProperty:)`
+    - `RealmCollection.max(ofProperty:)`
+    - `RealmCollection.average(ofProperty:)`
+    - `RealmCollection.sum(ofProperty:)`
```

Comment:
I think these should just be `min(of:)` etc. The `Property` is there to indicate what the string is and so isn't needed with a better type.